README.md

```diff
@@ -14,7 +19,9 @@ public function registerBundles()
 {
     $bundles = [
         // ...
+        new Doctrine\Bundle\DoctrineBundle\DoctrineBundle(), // if not already enabled
```

Comment:
is it here only because of the `ArrayCollection`? If yes, then no need to enable it